CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Fixed
 - Fixed issue with necessary columns from complex arguments dropped when interchanging dataframes [[#4324](https://github.com/plotly/plotly.py/pull/4324)]
+- Fixed issue with px.imshow failing when facet_col is an earlier dimension than animation_frame for xarrays [[#4330](https://github.com/plotly/plotly.py/issues/4330)]
+- Fixed issue with px.imshow failing when facet_col has string coordinates in xarrays [[#4329](https://github.com/plotly/plotly.py/issues/4329)]
```

Comment:
Would you create a new `## UNRELEASED` section above `## [5.16.1]` and move these there? They can also be combined and reference the PR, something like "Fixed two issues with px.imshow: [[#4330](https://github.com/plotly/plotly.py/issues/4330)], facet_col is an earlier dimension than animation_frame for xarrays, and [[#4329](https://github.com/plotly/plotly.py/issues/4329)], facet_col has string coordinates in xarrays [[#4331](https://github.com/plotly/plotly.py/pull/4331)]"